src/Setup.php

```diff
@@ -305,5 +307,23 @@ public function bcgov_block_theme_page_display() {
 
 	}
 
+	/**
+	 * Register ACF Pro Blocks.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	public function bcgov_block_theme_acf_init_block_types() {
+
+		// Check function exists.
+		if ( function_exists( 'acf_register_block_type' ) ) {
 
+			// Load CleanBC Blocks.
+			if ( defined( 'Bcgov\\Theme\\Block\\CLEANBC' ) && CLEANBC ) {
```

Comment:
I'm not sure why you made CLEANBC all uppercase, i feel like this is going against namepsace naming conventions.
I also feel like the main theme is being polluted with cleanbc, where it should be completely  separate.
I also think you could have benefitted from an autoloader, and could have got away from have includes and require_once all over the place.

Reply:
This is an error. I am about to push a fix for this check. All I can say is this should not have been pushed.